### PR TITLE
Adding external apps and Azure Communication Services

### PR DESCRIPTION
**Pull request by tophpalmer**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d06babf2d9b90fd4c9538487e772503ad79c618a/msteams-platform/overview.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8cae064c-1226-4c49-9eff-ec6235973f7d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/overview.md [hunk 1]
```diff
@@ -203,6 +203,13 @@ Looking for app development inspiration? Browse our list of real-world scenarios
    :::column-end:::
 :::row-end:::
 
+## Integrating Teams communication in an external app
+This page focuses on embedding your own experiences into Microsoft Teams. If you'd like to reverse this model and integrate Teams or other communication capabilities into your own app experience, check out [Azure Communication Services](https://docs.microsoft.com/azure/communication-services/overview).
+
+Azure Communication Services are cloud-based services with REST APIs and client library SDKs to help you integrate communication into your own custom applications. A [UI library](https://azure.github.io/communication-ui-library/) for desktop and mobile web browsers can help you quickly embed generic or Teams-styled components for chat and calling.
```
Comment (v-abirade) [inline msteams-platform/overview.md:209]: You can embed generic or Teams-styled components for calling and chatting with the help of [UI library](https://azure.github.io/communication-ui-library/).

**Review by v-abirade — COMMENTED** [review]
Please implement the change.

File: msteams-platform/overview.md [hunk 2]
```diff
@@ -203,6 +203,13 @@ Looking for app development inspiration? Browse our list of real-world scenarios
    :::column-end:::
 :::row-end:::
 
+## Integrating Teams communication in an external app
+This page focuses on embedding your own experiences into Microsoft Teams. If you'd like to reverse this model and integrate Teams or other communication capabilities into your own app experience, check out [Azure Communication Services](https://docs.microsoft.com/azure/communication-services/overview).
```
Comment (v-abirade) [inline msteams-platform/overview.md:207]: see [Azure Communication Services](https://docs.microsoft.com/azure/communication-services/overview).

**Review by v-abirade — CHANGES_REQUESTED** [review]
Please update the changes as suggested.

**Comment by v-abirade** [comment]
Hi @tophpalmer,

Please incorporate the changes as suggested.

**Review by v-abirade — CHANGES_REQUESTED** [review]
@tophpalmer,

Please make the changes.

**Comment by tophpalmer** [comment]
@v-abirade thank you for the feedback, changes are committed!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a6144b844419f3149de406da103205774f89524f/msteams-platform/overview.md) | 92 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d2fdd51c-dbf9-4a11-a93f-be81fe8cf8d3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dd05186303b7eb838cb26d4a000d4135bfe73dab/msteams-platform/overview.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9386ae68-d07e-47c3-9983-3efb974cd544) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1597567f20a6099f205ba4dd105409632969ade2/msteams-platform/overview.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63425617-1f3b-48bb-90a8-0970733ac71c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f912e7ad1332edae832498da5b37a633d5c44579/msteams-platform/overview.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be7558d6-61dc-4f6c-ba70-672b904646f2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approve

**Pull request merged**